package.json

```diff
@@ -23,7 +23,7 @@
   "dependencies": {
     "@4chain-ag/react-configuration": "^1.0.4",
     "@bsv/sdk": "^1.3.12",
-    "@bsv/spv-wallet-js-client": "^1.0.0-beta.30",
+    "@bsv/spv-wallet-js-client": "^1.0.0-beta.33",
```

Comment:
It should be:
`"@bsv/spv-wallet-js-client": "^1.0.0-beta.36",`
if we could try to bump to newest version.
It shouldn't be so much changes.